apps/address-service/domains/address/schema/LinkAddressAndSourceService.test.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Generated by `createservice address.LinkAddressAndSourceService --type mutations`
+ */
+
+const { faker } = require('@faker-js/faker')
+
+const { makeLoggedInAdminClient, expectToThrowGQLErrorToResult } = require('@open-condo/keystone/test.utils')
+
+const { ERRORS } = require('@address-service/domains/address/schema/LinkAddressAndSourceService')
+const { AddressSource, linkAddressAndSourceByTestClient, createTestAddress } = require('@address-service/domains/address/utils/testSchema')
+
+describe('LinkAddressAndSourceService', () => {
+    test('Should link with tin', async () => {
+        const tin = faker.random.numeric(12)
+        const source = `${faker.address.city()}${faker.random.alphaNumeric(8)}, ${faker.address.street()}, ${faker.random.alphaNumeric(8)}`
+        const admin = await makeLoggedInAdminClient()
+        const [{ id: address }] = await createTestAddress(admin)
+        const payload = { source, tin, address }
+        const [result] = await linkAddressAndSourceByTestClient(admin, payload)
+        expect(result).toMatchObject({
+            addressSourceId: expect.any(String),
+        })
+        const addressSource = await AddressSource.getOne(admin, { id: result.addressSourceId })
+        expect(addressSource.source).toEqual(expect.stringContaining('|helpers:'))
```

Comment:
Please add checking for address model connected
```suggestion
        expect(addressSource.source).toEqual(expect.stringContaining('|helpers:'))
        expect(addressSource.address.id).toEqual(address)
```